src/TeaAsAService/Tea.Core/Domain/User.cs

```diff
@@ -72,5 +80,42 @@ public static User CreateLocalDevUser()
                 LastBrewTimeUtc = DateTime.UtcNow
             };
         }
+
+        static bool ValidatePassword(string password)
+        {
+            const int MIN_LENGTH = 8;
+
+            if (password == null) throw new ArgumentNullException();
+
+            bool meetsLengthRequirements = password.Length >= MIN_LENGTH;

```

Comment:
We can just return out from here right away if the length check fail, there's no need to check the rest of it.